docs/src/man/split_apply_combine.md

```diff
@@ -1,26 +1,34 @@
-# The Split-Apply-Combine Strategy
+# Transforming data frames
```

Comment:
This is probably too general. Better keep something more specific to grouping, at least for now, and maybe later reorganise docs so that we present the `select`/`transform`/`combine` API elsewhere in detail outside of the context of grouping.

Reply:
OK - I have reverted.